src/app/modules/main/communities/tokens/models/token_owners_model.nim

```diff
@@ -83,7 +83,7 @@ QtObject:
       of ModelRole.WalletAddress:
         result = newQVariant(item.ownerDetails.address)
       of ModelRole.Amount:
-        result = newQVariant(item.amount)
+        result = newQVariant(item.amount.toString(10))
```

Comment:
hard to say what is better - plain number (which is very obvious for me in this case and easy to read) or named variable holding that value. How to call that constant, where to store it  - it may be needed in various files so one common would be needed. It entails additional imports... So I would opt for just plain number in this specific place as an exception from general rule of avoiding magic numbers ;)